docs/source/fitfunctions/DiffSphere.rst

```diff
@@ -38,17 +36,15 @@ condition\ `2 <http://apps.webofknowledge.com/InboundService.do?SID=4Bayo9ujffV3
 The roots of this set of equations are the numerical coefficients
 :math:`x_{n,l}`.
 
-The fit function DiffSphere has an elastic part, modelled by fitting
-function ElasticDiffSphere and an inelastic part, modelled by
-InelasticDiffSphere.
-
-When using InelasticDiffSphere, he value of Q can be obained either though the Q
-attribute or can be calucated from the input workspace using the WorkspaceIndex
-property. The value calculated using the workspace is used whenever the Q
-attibute is empty.
+The fit function DiffSphere has an elastic part modeled by fitting
+function `ElasticDiffSphere <ElasticDiffSphere>`__,
+and an inelastic part modeled by `InelasticDiffSphere <InelasticDiffSphere>`__.
```

Comment:
I'm not sure why but these links between the function pages don't work for me in the Qt help browser. For functions you can use links like 
```
:ref:`Convolution <func-Convolution>`
```
That should fix the issue and it's a more specific way of pointing to functions in the documentation.